app/services/verify_link_service.rb

```diff
@@ -26,7 +26,7 @@ def perform_request!
   def link_back_present?
     return false if @body.blank?
 
-    links = Nokogiri::HTML5(@body).css("a[rel~='me'],link[rel~='me']")
+    links = Nokogiri::HTML5(@body).xpath('//a[@rel] | //link[@rel]').select { |el| el.kwattr_values('rel').any? { |v| v.downcase == 'me' } }
```

Comment:
This is not the worst - but it'd be great if there was a purely xpath-query-native way to do this that nokogiri supported and we could get the full search done there instead of needing the ruby select.

Reply:
Done. I didn't realize that this kind of query occurs several places in the code, but it makes sense to make it a reusable component.